indra/newview/llvoavatar.cpp

```diff
@@ -8298,45 +8478,50 @@ BOOL LLVOAvatar::processFullyLoadedChange(bool loading)
         // models to snap into place, textures to get first packets, LODs to load.
         const F32 LOADED_DELAY = 1.f;
 
-        F32 delay = LOADED_DELAY;
```

Comment:
No way this mixed up merge here is going to be correct!

Reply:
Wow. Thanks!